shared/ui-components/src/modals/Modal.tsx

```diff
@@ -26,46 +32,97 @@ export const Modal: FunctionComponent<ModalProps> = ({
   fullScreen = false,
   onClose,
   children,
+  ariaDescription,
 }) => {
+  // append modal to the root of app
+  const modalRoot = typeof document !== "undefined" ? document.querySelector("#__next") : null
+  const el = typeof document !== "undefined" ? document.createElement("div") : null
```

Comment:
This should probably be wrapped in `useState` or similar, no? Right now every time this component re-renders it's going to create a new "div" element...

Reply:
Correct! I moved these elements to the useState. Good catch, thanks 👍